src/_guides/language/effective-dart/usage.md

```diff
@@ -452,71 +452,70 @@ possible avoid using `cast()` to "change" a collection's type.
 
 Prefer any of these options instead:
 
-#### Create it with the right type
+*   **Create it with the right type.** Change the code where the collection is
+    first created so that it has the right type.
 
-Change the code where the collection is first created so that it has the right
-type:
+*   **Cast the elements on access.** If you immediately iterate over the
+    collection, cast each element inside the iteration.
+
+*   **Eagerly cast using `List.from()`.** If you'll eventually access most of
+    the elements in the collection, and you don't need the object to be backed
+    by the original live object, convert it using `List.from()`:
+
+    The `cast()` method returns a lazy collection that checks the element type
+    on *every operation*. If you perform only a few operations on only a few
+    elements, that laziness can be good. But in many cases, the overhead of lazy
+    validation and of wrapping outweighs the benefits.
+
+Here are examples of each approach:
```

Comment:
I avoid using two colons in a row. Actually, let's delete this sentence and convert the headings into sentence-style intros. (Is there an important reason to have headings here?) The result would be something like this:

----------------------

...outweighs the benefits.

Here is example code for the first option, **create it with the right type**:
...
Here are examples for **cast each element on access**:
...
Here are examples for **cast eagerly using `List.from()`**:
...

These alternatives don’t always work....

----------------------

Reply:
@chalin can you think of a way to save vertical space here? These samples could be side-by-side, like diffs (except I suspect that diffs here wouldn't save space and might not be clear).

Reply:
(Do you have a preview somewhere by any chance? If not, I'll render and take a look tomorrow.)

Reply:
I'd vote to leave it as is for now. If we decide to tackle putting good/bad code excerpts side-by-side (for wide screens), we should do that throughout the Effective Dart pages. Do you want to create a separate issue to track this?

Reply:
I like your suggestion, Kathy.